### ISSUE TITLE
resolve-plugins-relative-to option, dont respect passed path, when linting directory has .eslintrc config

### ISSUE THREAD

**Issue by NikitaNaumenko** [eslint/eslint#12057]
**Tell us about your environment**

* **ESLint Version:** 6.1.0
* **Node Version:** 10.16.0
* **npm Version:** 6.9.0

**What parser (default, Babel-ESLint, etc.) are you using?**
default
**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
// eslint_test/.eslintrc.yml
---

extends:
  - airbnb
  - "plugin:jest/recommended"
  - "plugin:react/recommended"

parser: babel-eslint

env:
  browser: true
  node: true
  jest: true

rules:
  import/no-unresolved: 0
  import/extensions: 0
  react/prop-types: 0
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**
When i run eslint with options `--config` and `--resolve-plugins-relative-to`, on some directory which already have eslint config file, i've got an error

```bash
./eslint_test/node_modules/.bin/eslint ./test/index.js -c ./eslint_test/.eslintrc.yml --resolve-plugins-relative-to ./eslint_test
```

**What did you expect to happen?**
I expect eslint will be reslove plugins in path which i set, and use eslint config which i set in options

**What actually happened? Please include the actual, raw output from ESLint.**
i've got an error
```
Oops! Something went wrong! :(

ESLint: 6.1.0.

ESLint couldn't find the config "airbnb" to extend from. Please check that the name of the config is correct.

The config "airbnb" was referenced from the config file in "/Users/nikitanaumenko/Workspace/test/.eslintrc.yml".

If you still have problems, please stop by https://gitter.im/eslint/eslint to chat with the team.
```

**Are you willing to submit a pull request to fix this bug?**
maybe

**Comment by kaicataldo** [comment]
Closing this as it looks like the question has been answered. Please feel free to stop by [our chat](https://gitter.im/eslint/eslint) if you run into any issues.
